src/lib/system/utils.dart

```diff
@@ -26,6 +26,7 @@ Future<Color?> getBackgroundColor(ImageProvider image) async {
     paletteGenerator.darkVibrantColor?.color,
     paletteGenerator.lightMutedColor?.color,
     paletteGenerator.dominantColor?.color,
+    Colors.black,
   ];
 
   return colors.firstWhere((c) => c != null);
```

Comment:
The rest of the code can handle a null result. The better fix would be to use `firstWhereOrNull`, I think.

Reply:
You're right. I've updated the PR